src/index.css

```diff
@@ -1,13 +1,60 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+  font-family: "Roboto", -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
+a {
+  color: #000000;
+}
+
+input {
+  border-color: #9B9B9B;
+}
+
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+#signInContainer {
+  height: 50vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#signUpContainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#signInForm {
+  background-color: #E7E7E7;
```

Comment:
We could maybe create a new file file called `colours.ts` which would have different colours exported.

So for example the file could be

`
export const colours = {
 greyLight: #E7E7E7
}
`

and then we could just use it here `background-color: colours.greyLight`

Reply:
Oh yea this might not be possible with pure css files. Nevermind then